devel/fatal/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        facebook fatal 2023.01.30.00 v
+revision            0
+categories          devel
+license             BSD
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Library for fast prototyping software in modern C++
+long_description    Fatal is a library for fast prototyping software in modern C++. \
+                    It provides facilities to enhance the expressive power of C++. \
+                    The library is heavily based on template meta-programming, \
+                    while keeping the complexity under-the-hood.
+checksums           rmd160  cea1204d92930efee0da93c1444ebf53e60c6471 \
+                    sha256  40b7c4a644ad663232d255fffbdfd87be09872304165c67f46cc93c052319532 \
+                    size    651285
+github.tarball_from releases
+distname            ${name}-v${version}
+extract.mkdir       yes
```

Comment:
why do the default vales for "distname" and "extract.mkdir" not work - the almost always do...

Reply:
@reneeotten `distname` is non-standard:
https://github.com/facebook/fatal/archive/refs/tags/v2023.01.02.00.tar.gz
https://github.com/facebook/fatal/releases/download/v2023.03.13.00/fatal-v2023.03.13.00.tar.gz
Default would be without -v part, and it fails to work.